lib/nimbase.h

```diff
@@ -177,12 +178,13 @@ __AVR__
 #  define N_THISCALL_PTR(rettype, name) rettype (__thiscall *name)
 #  define N_SAFECALL_PTR(rettype, name) rettype (__stdcall *name)
 
-#  ifdef __cplusplus
-#    define N_LIB_EXPORT  NIM_EXTERNC __declspec(dllexport)
```

Comment:
It's unnecessary to have a define which is the same as the other branch. So I removed this `__cplusplus` define to simplify code.